runtime/src/main/java/org/corfudb/runtime/collections/Table.java

```diff
@@ -83,7 +84,7 @@ public Table(@Nonnull final String namespace,
                 .setTypeToken(CorfuTable.<K, CorfuRecord<V, M>>getTableType())
                 .setStreamName(this.fullyQualifiedTableName)
                 .setSerializer(serializer)
-                .setArguments(new ProtobufIndexer(valueSchema))
+                .setArguments(new ProtobufIndexer(valueSchema), streamingMap)
```

Comment:
Creating `StreamingMap` should happen via CorfuTable, look at my comment above about the leak.

Reply:
Same thing. Supplier<T> should fix it.